drop-transfer/src/utils.rs

```diff
@@ -149,6 +150,14 @@ pub fn to_kb(bytes: u64) -> i32 {
     (bytes as f64 / 1024.0).ceil() as i32
 }
 
+/// Get the current system timestamp in milliseconds, used for event timestamps
+pub fn current_timestamp() -> u64 {
+    SystemTime::now()
+        .duration_since(SystemTime::UNIX_EPOCH)
+        .expect("Failed to retrieve system time")
```

Comment:
Um, it probably would never fail but I'd go for `unwrap_or_default()` just in case

Reply:
BTW, why this is in the `drop-transfer` crate? It's used only in the `norddrop` crate

Reply:
Made sense for me to put it here, since `norddrop` is an extension of `drop-transfer`, and it already has a `utils` file that conveniently houses utils, in addition to it being a good spot if we decided to move the timestamps to being created in internal events, as opposed to what we do now

Reply:
Though i wouldn't be against moving it if you think it makes more sense to be in `norddrop` instead

Reply:
I mean things should be defined near the usage, though it's not too much of a deal.
However, it seems logical to keep it it here and use the timestamp on the internal events, or define it in the norddrop and use it only there. The mix of these two is a bit odd to me

Reply:
In any case, it's your call